qiskit_nature/operators/second_quantization/vibrational_op_utils/vibrational_labels_validator.py

```diff
@@ -29,7 +32,8 @@ def _validate_vibrational_labels(vibrational_labels: List[Tuple[str, complex]],
             * operators in each label are sorted in the decreasing order of modes and modals,
             if both are equal then '+' comes before '-' (i.e. they are normal ordered),
             * for each `+` operator in each label, there is a corresponding '-' operator
-            acting on the same mode (i.e. the number of particles is preserved per mode).
+            acting on the same mode (i.e. the number of particles is preserved per mode) [this last
+            case will only log a warning]
```

Comment:
What does this mean - A warning is logged is particles are preserved (or is the operators are such that this is not the case that they are preserved). I was trying to read that in conjunction with the warning below but that just says the number mismatches which leads me to believe that the warning is emitted if thats not the last case and particles are not preserveed

Reply:
I clarified this docstring.